UnitystationLauncher/ViewModels/ChangelogViewModel.cs

```diff
@@ -1,41 +1,52 @@
+using System.Collections.Generic;
 using ReactiveUI;
 using Octokit;
 using System.Collections.ObjectModel;
 using System.Reactive.Concurrency;
+using System.Threading;
 using System.Threading.Tasks;
+using Serilog;
 
 namespace UnitystationLauncher.ViewModels
 {
     public class ChangelogViewModel : ViewModelBase
     {
-        readonly GitHubClient _client;
-        ObservableCollection<PullRequestViewModel> _pullRequests = new ObservableCollection<PullRequestViewModel>();
+        private readonly GitHubClient _client;
 
-        public ObservableCollection<PullRequestViewModel> PullRequests
-        {
-            get => _pullRequests;
-            set => this.RaiseAndSetIfChanged(ref _pullRequests, value);
-        }
+        ObservableCollection<PullRequestViewModel> PullRequests { get; }
```

Comment:
```suggestion
        private ObservableCollection<PullRequestViewModel> PullRequests { get; }
```

I understand this didn't have it before, but please prefer to use explicit access modifier.

Reply:
No worries, please do tell me about these if you see them. Getting the code to be more standardized is always good.